executor/show_test.go

```diff
@@ -361,3 +361,33 @@ func (s *testSuite) TestIssue3641(c *C) {
 	_, err = tk.Exec("show table status;")
 	c.Assert(err.Error(), Equals, plan.ErrNoDB.Error())
 }
+
+// TestShow2 is moved from session_test
+func (s *testSuite) TestShow2(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+
+	tk.MustExec("set global autocommit=1")
+	tk.MustQuery("show global variables where variable_name = 'autocommit'").Check(testkit.Rows("autocommit 1"))
```

Comment:
Why did it change from `ON` to `1`?

Reply:
This is changed in other test, I have fixed it.